internal/controllers/s3userclaim/handler.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package s3userclaim
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ceph/go-ceph/rgw/admin"
+	"github.com/go-logr/logr"
```

Comment:
Using this logger is forced by kubernetes?

Reply:
No. It uses `logr.Logger` which allows using any compatible implementations such as [`zapr`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/log/zap).

If you see advantages in using a different logging library, I'm happy to do so.

Reply:
Awesome, we can use `zapr` in future :see_no_evil: